dist/5655-schema.json

```
@@ -261,24 +261,68 @@
         "string",
         "null"
       ]
+    },
+    "ssn": {
+      "type": "string",
+      "pattern": "^[0-9]{9}$"
+    },
+    "employmentHistory": {
+      "type": "array",
+      "items": {
+        "type": "object",
+        "properties": {
+          "veteranORSpouse": {
```

```suggestion
          "veteranOrSpouse": {
```
Should this be camelCased?

---

It should be, but our service partners at the DMC have opted to represent it using this format, I am trying to stay consistent with the payload they are ultimately expecting during the submission of the form. I will talk to them about changing it to the format you are suggesting, as it is what I prefer as well, but for now it will have to be like it is.